### PR TITLE
[Fix] control_service doesn't manage Wazuh daemons correctly

### LINKED ISSUE #1316: Auth integration tests error
image

There is an error in our auth integration tests when running on the master branch.

The problem is that the `configure_sockets_environment` function calls to `check_daemon_status` after stopping wazuh and this function has a timeout (by default) of 10 seconds. As some daemons (in this case, clusters) have not stopped in these 10 seconds. The test fails before even start.

We need to increase the timeout of `check_daemon_status` function and check fi this sovles the problem.

### PR DESCRIPTION
**Pull request by mdengra**
|Related issue|
|---|
|Closes #1316|

## Description

This PR fixes the problem with the `control_service` function in which it is not able to handle Wazuh daemons running through Python, causing some tests, such as `test_authd` to fail when trying to terminate the `wazuh-clusterd` daemon, which runs through a Python module and not as a standalone process.

### CODE REVIEW

**Review by spothound — APPROVED** [review]
Awesome work @mdengra! Thank you very much.

**Comment by spothound** [comment]
## Jenkins testing

Example of failed checks: https://ci.wazuh.info/job/Test_integration/3796/
Replay with the fix: https://ci.wazuh.info/job/Test_integration/3874/console

File: deps/wazuh_testing/wazuh_testing/tools/monitoring.py [hunk 1]
```diff
@@ -779,8 +779,8 @@ def run(self, *args):
 
         # Give proper permissions to socket
         if isinstance(self.listener_socket_address, str):
-            uid = pwd.getpwnam('wazuh').pw_uid
-            gid = grp.getgrnam('wazuh').gr_gid
+            uid = pwd.getpwnam('ossec').pw_uid
+            gid = grp.getgrnam('ossec').gr_gid
```
Comment (BraulioV) [inline deps/wazuh_testing/wazuh_testing/tools/monitoring.py:783]: Revert this change

**Review by BraulioV — CHANGES_REQUESTED** [review]

File: deps/wazuh_testing/wazuh_testing/tools/monitoring.py (hunk 1 above)
Comment (mdengra) [reply deps/wazuh_testing/wazuh_testing/tools/monitoring.py]: Done in: https://github.com/wazuh/wazuh-qa/pull/1345/commits/ba4f4aa7304d2cf27ce9776b834e87e1cbeed8c4

**Review by BraulioV — APPROVED** [review]
